GameData/ROCapsules/Assets/DECQ/SoyuzMS/Props/Neptune/prop.cfg

```diff
@@ -1,53 +1,57 @@
-PROP
+PROP:NEEDS[RasterPropMonitor]
 {
-  name = NEPTUNE4020
-
- MODULE
-  {
-	name = RasterPropMonitor
-
-	screenTransform = INPU1_LCD 
-									// Name of transform for the screen surface.
-	fontTransform = BobCatind/HOME2/Props/Neptune/NEPTUNE_font 
-        fontLetterWidth = 16
-        fontLetterHeight = 32
-	textureLayerID = _Emissive
-
-        emptyColor = 0,0,0,0
-    									// Name of texture layer to work with on the screen surface.
-	screenWidth = 60                      									// Screen width in letters.
-	screenHeight = 20
-                     									// Screen height in lines.
-	screenPixelWidth = 960               									// Width of the generated screen texture in pixels.
-	screenPixelHeight = 640               									// Height of the generated screen texture in pixels.
-	               									// Height of a font character.
-	cameraAspect = 2
-
-                refreshDrawRate = 2
+	name = NEPTUNE4020
+
+	MODULE
+	{
+		name = RasterPropMonitor
+
+		screenTransform = INPU1_LCD
+
+		// Name of transform for the screen surface.
+		fontTransform = BobCatind/HOME2/Props/Neptune/NEPTUNE_font
```

Comment:
I don't think these are valid as expected paths?  (Bobcat Industries is a separate mod, right?)